rosidl_generator_py/package.xml

```diff
@@ -6,17 +6,37 @@
   <maintainer email="esteve@osrfoundation.org">Esteve Fernandez</maintainer>
   <license>Apache License 2.0</license>
 
+  <build_depend>python_cmake_module</build_depend>
+  <build_depend>rmw</build_depend>
+  <build_depend>rmw_implementation</build_depend>
+  <build_depend>rosidl_generator_c</build_depend>
+
+  <build_export_depend>python_cmake_module</build_export_depend>
```

Comment:
Since it is also a `buildtool_depend` and targeting the build platform this is not needed.